packages/maker.js/src/core/svg.ts

```diff
@@ -477,7 +477,7 @@ namespace MakerJs.exporter {
 
         //begin svg output
 
-        var svgAttrs: IXmlTagAttrs;
+        var svgAttrs: IXmlTagAttrs = {};
```

Comment:
Perhaps you can just put the attribute here.`

Reply:
I considered a few ways to make this change, but was I trying to leave as much code intact.  I went with this approach because by initializing it here and then adding the attribute on line 496, I was able to not have to change line 492 where it is re-assigned and was also able to leave the `extendObject` merge of passed in options on line 497 alone.